src/app/services/components/ServiceInsightDetails.vue

```diff
@@ -62,7 +58,6 @@ import { useRoute } from 'vue-router'
 import { ServiceInsight } from '@/types/index.d'
 import { stripTimes } from '@/utilities/helpers'
 import { STATUS } from '@/constants'
-import EntityURLControl from '@/app/common/EntityURLControl.vue'
 import YamlView from '@/app/common/YamlView.vue'
 
 const route = useRoute()
```

Comment:
Suggestion: Please remove the no longer needed definition of route and its related import.